src/lib/uni/src/central.ts

```diff
@@ -0,0 +1,175 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {Inject, Injectable} from '@angular/core';
+import {Directionality} from '@angular/cdk/bidi';
+import {MediaMatcher} from '@angular/cdk/layout';
+
+import {UnifiedDirective} from './unified';
+import {BPS, Breakpoint} from './breakpoint';
+import {Tag, ValuePriority} from './tags/tag';
+import {TAGS} from './tags/tags';
+import {DIR_KEY, KEY_DELIMITER, NO_VALUE, PARENT_KEY, SELF_KEY, STYLE_KEY} from './constants';
+
+
+/**
+ * GrandCentral is a switchyard for all of the various Layout directives
+ * registered in an application. It is the single source of truth for all of
+ * the layout changes that occur in an application. For instance, any changes
+ * to the browser state via registered media queries are monitored and updated
+ * in this service. The directives themselves simply store their respective values
+ * for each of the media states.
+ */
+@Injectable({providedIn: 'root'})
+export class GrandCentral {
```

Comment:
`GrandCentral` is semi-obvious for Americans who live on the East Coast, but I'm not sure it's as universally well-known as we might want. Would something like `Orchestrator` or `Conductor` (I think that this could have minor collisions with other meanings of the word) be more widely understood?